docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -353,115 +479,72 @@ export const pageQuery = graphql`
 
 Further examples can be found in the [Gatsby Image docs](https://github.com/gatsbyjs/gatsby/tree/d0e29272ed7b009dae18d35d41a45e700cdcab0d/packages/gatsby-image).
 
-## Manually specify PostCSS plugins
-
-Gatsby v2 removed `postcss-cssnext` and `postcss-import` from the default PostCSS setup.
-
-To have the same configuration that you had in v1, you should use [`gatsby-plugin-postcss`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-postcss) and follow the recommended migration path below.
+### Delete Nodes API Deprecated
 
-### 1. Install the dependencies
+Stub content. PRs welcome!
 
-`npm install --save gatsby-plugin-postcss postcss-import postcss-cssnext postcss-browser-reporter postcss-reporter`
+## Other Changes Worth Noting
 
-**NOTE**: `postcss-cssnext` is [deprecated](https://moox.io/blog/deprecating-cssnext/) and it is better to use `postcss-preset-env` now.
+### Explicit query names no longer required
 
-### 2. Include `gatsby-plugin-postcss` in your `gatsby-config.js` file
+Gatsby v2 doesn't require explicit query names. You can skip them now:
 
-```js
-// in gatsby-config.js
-plugins: [`gatsby-plugin-postcss`],
+```diff
+export const query = graphql`
+-  query ThisIsExplicitQueryName($slug: String!) {
++  query($slug: String!) {
+    markdownRemark(fields: { slug: { eq: $slug } }) {
+      html
+      frontmatter {
+        title
+      }
+    }
+  }
 ```
 
-### 3. Include PostCSS plugins in your `postcss.config.js` file
-
-```js
-// in postcss.config.js
-const postcssImport = require(`postcss-import`);
-const postcssCssNext = require(`postcss-cssnext`);
-const postcssBrowserReporter = require(`postcss-browser-reporter`);
-const postcssReporter = require(`postcss-reporter`);
+You can also skip the `query` keyword if you don't use query variables:
 
-module.exports = () => ({
-  plugins: [
-    postcssImport(),
-    postcssCssNext(),
-    postcssBrowserReporter(),
-    postcssReporter(),
-  ],
-})
+```diff
+export const query = graphql`
+-  query ThisIsAnotherExplicitQueryName {
++  {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
 ```
 
-## Convert to either pure CommonJS or pure ES6
-
-Gatsby v2 uses babel 7 which is stricter about parsing files with mixed JS styles.
-
-ES6 Modules are ok:
+This isn't a breaking change. Queries with explicit names will continue to work as they did in v1.
 
-```js
-// GOOD: ES modules syntax works
-import foo from "foo"
-export default foo
-```
+### Remove inlined CSS in `html.js`
 
-CommonJS is ok:
+Gatsby v2 will automatically inline your CSS. You can remove any custom CSS inlining from your custom `html.js`.
```

Comment:
Also suggest that if they're doing anything special with their html.js, they can just remove it now and rely on the default.